src/transformers/utils/import_utils.py

```diff
@@ -614,6 +614,14 @@ def is_flash_attn_2_available():
     return _flash_attn_2_available and torch.cuda.is_available()
 
 
+def is_flash_attn_available():
+    warnings.warn(
```

Comment:
We're trying to move towards using `logger` for warnings so that users have more control over the verbosity of their logs 

```suggestion
    logger.warning(
```